src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator machine started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+
+        System.out.println("Excavator machines stopped working");

```

Comment:
The message in the 'stopWork' method should be more informative and indicate that the 'Excavator' has stopped working. Also, there is a typographical error, it should be 'Excavator machine' instead of 'Excavator machines'. For example, 'Excavator has stopped its work.'